src/tests/ftest/util/agent_utils.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, path="", yaml_cfg=None, timeout=30):
 
         # Command line parameters:
         # -d, --debug        Enable debug output
-        # -j, --json         Enable JSON output
+        # -J, --json-logging Enable JSON logging
```

Comment:
Shouldn't line 78  also reflect this change - at least for consistency:
```
self.json = FormattedParameter("--json-logging", False)
```

Reply:
Ah, yep. Good catch, missed these.